src/app/dashboard/components/chart/index.tsx

```diff
@@ -21,11 +21,14 @@ type Props = {
 
 const StockChart: React.FC<Props> = ({ data }) => {
   const theme = useTheme();
-  const isAggregate = data[0].low !== undefined && data[0].high !== undefined;
+  const isAggregate =
+    data.length !== 0 &&
+    data[0].low !== undefined &&
+    data[0].high !== undefined;
```

Comment:
Możemy w sumie przestać wyświetlać wykres jeżeli nie ma żadnych danych dla niego - ktoś ustawił złe filtry

Przenieś to tylko pod `useChartParams` bo inaczej będzie problem z hookami

```suggestion
  if (data.length === 0)
    return <Typography>No data - please adjust your filters.</Typography>;

  const isAggregate =
    data[0].low !== undefined &&
    data[0].high !== undefined;
```

Reply:
Poprawiłem kod, poza "Przenieś to tylko pod useChartParams bo inaczej będzie problem z hookami"
Jakie problemy z hookami masz na myśli?

Reply:
https://legacy.reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

"Don’t call Hooks inside loops, conditions, or nested functions. Instead, always use Hooks at the top level of your React function, before any early returns."

więc `return <Typography>....` jest właśnie zaliczany jako "early return"

Reply:
ok, poprawiłem